src/protobuf/mod.rs

```diff
@@ -0,0 +1,4 @@
+// SPDX-License-Identifier: Apache-2.0
+
+//! This is a placeholder file for the Rust stuctures generated from
+//! aesm-proto.proto during build.
```

Comment:
```suggestion
//! Provides all generated protobuf modules

include!(concat!(env!("OUT_DIR"), "/protos/mod.rs"));
```

This then includes the generated protobuf modules.

You will then have to use `crate::protobuf::aesm_proto`.